pkgs/development/php-packages/openswoole/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, buildPecl, php, valgrind, pcre2 }:
+
+buildPecl {
+  pname = "openswoole";
+
+  version = "4.11.1";
+   sha256 = "sha256-Rhoa4ny86dwB3e86/1W30AlDGRUDYjK8RusquKF5Izg=";
+
+  buildInputs = [ pcre2 ] ++ lib.optionals (!stdenv.isDarwin) [ valgrind ];
+  internalDeps = lib.optionals (lib.versionOlder php.version "7.4") [ php.extensions.hash ];
+
+  doCheck = true;
```

Comment:
Not sure it's worth to... run the tests if it skips... all of them? Was this the thing that was "fixed" from upstream? Just ignore running them all?

```
=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :    0
Exts tested     :   12
---------------------------------------------------------------------

Number of tests : 1204                 0
Tests skipped   : 1204 (100.0%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :    0 (  0.0%) (  0.0%)
Tests passed    :    0 (  0.0%) (  0.0%)
---------------------------------------------------------------------
Time taken      :   17 seconds
=====================================================================
```

Reply:
@etu  Relayed the question to the developers, will get back to you.

Reply:
Yes,  the tests requiring an environment to run; the library is well tested in our CI system. I have removed `doCheck = true;`